tools/cli/admin.go

```diff
@@ -388,6 +388,10 @@ func newAdminDomainCommands() []cli.Command {
 					Name:  FlagAllWithAlias,
 					Usage: "List all domains, by default only domains in REGISTERED status are listed",
 				},
+				cli.BoolFlag{
+					Name: FlagDeprecatedWithAlias,
+					Usage: "List deprecated domains, by default only domains in REGISTERED status are listed",
```

Comment:
nit: maybe "List only deprecated domain"? I thought it will list deprecated domains in addition to registered domains at first glance.